### PR TITLE
feat(datetime-picker): Add range picker

### PR DESCRIPTION
**Pull request by dabeng**
## Description
This PR is targeted to issue #1078 
@mcarrano @matthewcarleton @nding-anges @dgutride  Could you review this PR whenever you got the chance?
## Changes

Introduce new test page for Date and Time based on [bootstrap datetimepicker](https://eonasdan.github.io/bootstrap-datetimepicker/).

## Link to rawgit and/or image

Here's the [Demo](https://rawgit.com/dabeng/patternfly/date-range-dist/dist/tests/date-and-time.html#!) page.

### CODE REVIEW

**Comment by mcarrano** [comment]
Looks good @dabeng . Just one question from a design standpoint. I expected either clicking on the field or clicking on the button to the right of the field to bring up the picker. Right now only the field is interactive.

@matthewcarleton and @jgiardino , would be great to get you guys to also review from a code standpoint.

**Comment by dabeng** [comment]
Yes. @mcarrano I have sent patches to allow users to trigger calendar by clicking input and icon button.

**Comment by amarie401** [comment]
LGTM from the code/js standpoint.

File: src/less/time-picker.less [hunk 1]
```diff
@@ -2,6 +2,21 @@
 // Time Picker
 // --------------------------------------------------
 .bootstrap-datetimepicker-widget {
+  .today-button-text:before {
+    content: "Today";
```
Comment (jeff-phillips-18) [inline src/less/time-picker.less]: This should be named something like `today-button-pf`. Internationalized applications will need to override this and may go back to using an icon if necessary.

**Review by jeff-phillips-18 — APPROVED** [review]

**Review by amarie401 — APPROVED** [review]

**Pull request merged**